docs/index.rst

```diff
@@ -56,10 +56,15 @@ variety of platforms.
 
     cloud/emr
 
+.. toctree::
+    :caption: Demo notebooks
+    :maxdepth: 2
+
```

Comment:
comment how these notebooks were generated

Reply:
better yet, maybe add a "make update_docs" to mango-python which copies the notebooks over to docs

Reply:
Done, `make update_docs` in mango-python now uses nbconvert to run notebooks and place output in docs/notebooks for readthedocs